src/govuk/components/error-summary/error-summary.yaml

```
@@ -10,11 +10,11 @@ params:
 - name: descriptionText
   type: string
   required: false
-  description: If `descriptionHtml` is set, this is not required. Text to use for the description of the errors. If `descriptionHtml` is provided,  `descriptionText` will be ignored.
+  description: Text to use for the description of the errors. If `descriptionHtml` is provided,  `descriptionText` will be ignored.
 - name: descriptionHtml
   type: string
-  required: true
-  description: If `descriptionText` is set, this is not required. HTML to use for the description of the errors. If `descriptionHtml` is provided, `descriptionText` will be ignored.
+  required: false
+  description: HTML to use for the description of the errors. If `descriptionHtml` is provided, `descriptionText` will be ignored.
```

```suggestion
  description: HTML to use for the description of the errors. If you set `descriptionText`, the component will ignore `descriptionHTML`.
```
Same tweak as above. I think the original also accidentally duplicated the `descriptionText` descripton - I've reversed it to say that if you use *this* variable, the other will be ignored.

---

I think the original was correct. If you've passed both `descriptionHTML` and `descriptionText`, `descriptionHTML` will always 'win' and `descriptionText` will always be ignored.

---

Ah sorry - I've tweaked my suggestion accordingly.